src/gui/static/src/app/components/pages/force-skywallet-wipe/force-skywallet-wipe.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { FormBuilder } from '@angular/forms';
+import { MatDialog, MatDialogConfig } from '@angular/material/dialog';
+
+import { ChildHwDialogParams } from '../../layout/hardware-wallet/hw-options-dialog/hw-options-dialog.component';
+import { HwWipeDialogComponent } from '../../layout/hardware-wallet/hw-wipe-dialog/hw-wipe-dialog.component';
+
+/**
+ * Allows wipe a Skywallet. It is mainly for restoring the device when the user is not able to
+ * use it because the operations are always cancelled for inactivity.
+ */
+@Component({
+  selector: 'app-force-skywallet-wipe',
+  templateUrl: './force-skywallet-wipe.component.html',
+  styleUrls: ['./force-skywallet-wipe.component.scss'],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163601)
```suggestion
  styleUrls: ["./force-skywallet-wipe.component.scss"],
```